core/dbt/lib.py

```diff
@@ -74,10 +81,6 @@ def get_dbt_config(project_dir, args=None, single_threaded=False):
 
     profile_name = getattr(args, "profile", None)
 
-    profile_name = getattr(args, "profile", None)
-
-    profile_name = getattr(args, "profile", None)
-
     runtime_args = RuntimeArgs(
```

Comment:
:)
:)
:)

Reply:
Once we have a better interface this whole file should be gone